### PR TITLE
review add-nodes

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

File: en/installconfig-add-nodes.md [hunk 1]
```diff
@@ -40,67 +39,53 @@ A node configured to boot via PXE will:
...
-sudo mkdir /home/maas
-sudo chown maas:maas /home/maas
 sudo chsh -s /bin/bash maas
+sudo su - maas
+ssh-keygen -f ~/.ssh/id_rsa -N ''
 ```
 
-Become the 'maas' user and generate a keypair with the private key having a
-null passphrase:
+Add the public key to `/ubuntu/.ssh/authorized_keys` on the KVM host:
```
Comment (degville) [inline en/installconfig-add-nodes.md]: I know this is inherited from the old docs, but I guess this should be /home/<USER>/.ssh/authorized_keys on the KVM host, where <USER> is the user managing the KVM guests.

Comment (pmatulis) [inline en/installconfig-add-nodes.md]: whoops!

**Comment by pmatulis** [comment]
@degville  Correction made, along with other minor improvements.

**Comment by degville** [comment]
Excellent - thanks Peter.

**Pull request merged**